### PR TITLE
web share API edge support is 81 or later. (windows only.)

### LINKED ISSUE #5566: web share API edge support is 81 or later. (windows only.)
https://developer.microsoft.com/en-us/microsoft-edge/status/webshareapi/

### PR DESCRIPTION
**Pull request by juner**
Fixes #5566

### CODE REVIEW

**Comment by atjn** [comment]
Wow yeah, you're right! It's really interesting to see a new feature land i Edge before Chrome. I guess we'll be seeing more of that now.

File: features-json/web-share.json [hunk 1]
```diff
@@ -49,11 +49,11 @@
       "16":"n",
       "17":"n",
       "18":"n",
-      "79":"n",
-      "80":"n",
-      "81":"n",
-      "83":"n",
-      "84":"n"
+      "79":"y",
+      "80":"y",
```
Comment (atjn) [inline features-json/web-share.json:53]:
```suggestion
      "79":"n",
      "80":"n",
```

The feature was added in Edge 81:

image

image

**Comment by juner** [comment]
thank you.

**Comment by atjn** [comment]
You're most welcome :)

Yesterday i forgot to check for Mac compatibility:

image

It seems like you will have to mark the feature "Partially Supported" (`a`) and add a note explaining that support is only for Windows. I was not able to test for support on older Windows versions, but i think just saying "Windows" is fine.

Sorry about that :/

**Comment by juner** [comment]
@atjn Is it like this?

**Comment by Fyrd** [comment]
Thanks!

**Pull request merged**